app/utils/Format.scala

```diff
@@ -27,8 +28,9 @@ object Format {
   def dateFormatted(date: LocalDate): String     = date.format(dateFormatter)
   def dateFormatted(date: LocalDateTime): String = date.format(dateFormatter)
 
-  val timeFormatter: DateTimeFormatter           = DateTimeFormatter.ofPattern("HHMM")
+  val timeFormatter: DateTimeFormatter           = DateTimeFormatter.ofPattern("HHmm")
   def timeFormatted(date: LocalDateTime): String = date.format(timeFormatter)
+  def timeFormatted(date: LocalTime): String     = date.format(timeFormatter)
```

Comment:
should this be an input of time rather than date?

Reply:
Do you mean the name of the parameter? In which case yes